packages/docs/src/components/code-block/index.js

```diff
@@ -3,12 +3,14 @@ import {h} from 'preact';
 import cx from 'classnames';
 import hljs from 'highlight.js/lib/highlight';
 import javascript from 'highlight.js/lib/languages/javascript';
+import typescript from 'highlight.js/lib/languages/typescript';
 import htmlbars from 'highlight.js/lib/languages/htmlbars';
 import json from 'highlight.js/lib/languages/json';
 import xml from 'highlight.js/lib/languages/xml';
+import python from 'highlight.js/lib/languages/python';
```

Comment:
This is a very one off use case, do we know how bug is this? If big can we load this lazily only for that component?

Reply:
I have to admit it's may be a bit wast full I maybe wanted to write a own "language" I only need the hashtag comment and that should ba a one-liner

Reply:
Added that simple language